### PR TITLE
Minor updates to landing page text

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: app/views/home/index.haml [hunk 1]
```diff
@@ -86,7 +86,7 @@
           Write code, submit a pull request, get review from Hound.
         %p
           Hound integrates with your existing workflow by reviewing and commenting
-          on code the moment a GitHub pull request is opened.
+          on code the moment a GitHub pull request is opened or udpated.
```
Comment (salbertson) [inline app/views/home/index.haml]: Does this make more sense?

Comment (teoljungberg) [inline app/views/home/index.haml]: `updated`

Comment (salbertson) [inline app/views/home/index.haml]: Ha, yes.

Comment (teoljungberg) [inline app/views/home/index.haml]: "on code` reads a bit off. Any ideas on how to make it clearer?

**Comment by croaky** [comment]
Looks good

**Pull request merged**